lib/ImageInfo.cpp

```diff
@@ -196,7 +196,7 @@ namespace CIL {
         }
         double allowed_error_percentage = 0;
         if (img1.imageType() == ImageType::JPEG)
-            allowed_error_percentage = 0.03;
+            allowed_error_percentage = 1;
```

Comment:
does JPEG need this much error percentage now?

Reply:
apparently yes. Now I am calculating error of a component as follows: 
```c++
double err = std:abs(static_cast<int>(data1(i, j, k)) - static_cast<int>(data2(i, j, k)))/255
```